cypress/objects/creatTask.js

```diff
@@ -0,0 +1,23 @@
+class CreatTask {
```

Comment:
CreateTask could live together with the TaskPage objects as those screens represent same page - Tasks CRUD